styles/styles.css

```diff
@@ -281,3 +282,69 @@ body {
   font-size: 12px;
   margin: 20px 0 72px 0;
 }
+
+.contact-form {
+  margin: 60px 24px 0 24px;
+}
+
+.contact-form form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact-form-title {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.name,
+.email,
+.message {
+  width: 100%;
+  margin-top: 24px;
+  border: 1px solid #d0d9d4;
+  padding: 15px;
+  box-sizing: border-box;
+}
+
+.message {
+  height: 114px;
+  resize: none;
+}
+
+.name::placeholder,
+.email::placeholder,
+.message::placeholder {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+}
+
+footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+footer .social-media {
+  margin-top: 40px;
+}
+
+.myemail {
+  font-weight: 400;
+  font-size: 20px;
+  color: #42526e;
+  margin-top: 18px;
+}
+
+.indicator2 {
+  height: 8px;
+  width: 150px;
+  margin-top: 320px;
+  background: #e5e5e5;
+  border-radius: 24px;
+}
```

Comment:
In order for your project to match the Figma design, 

- [ ] Please change the colour of it;
- [ ] Kindly, give it a small bottom margin too;

Figma design:
![image](https://user-images.githubusercontent.com/83596135/160712899-bb2b92ae-cabd-492e-bdc0-3011c8f285c8.png)


Your design:
![image](https://user-images.githubusercontent.com/83596135/160712944-a06f54db-8312-4cbd-9d8d-a193f868ac35.png)